core/errors/packager.h

```diff
@@ -28,5 +28,6 @@ constexpr ErrorClass UsedTestOnlyName{3720, StrictLevel::False};
 constexpr ErrorClass InvalidExport{3721, StrictLevel::False};
 // constexpr ErrorClass ExportingTypeAlias{3722, StrictLevel::False};
 constexpr ErrorClass ImportNotVisible{3723, StrictLevel::False};
+constexpr ErrorClass PackageIssues{3719, StrictLevel::False};
```

Comment:
Don't re-use numbers

```suggestion
constexpr ErrorClass PackageIssues{3724, StrictLevel::False};
```